### PR TITLE
Setup/flow

### PR DESCRIPTION
**Pull request by hmduerinck**
fix dependency versions
add workflow with actions

### CODE REVIEW

File: .github/workflows/github-actions-nafigator.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+name: Python package
+
+on:
+  push:
+    branches: [setup/flow]
```
Comment (royklip) [inline .github/workflows/github-actions-nafigator.yml:5]: This is a specific branch (for testing probably). Can it be removed?

File: .github/workflows/github-actions-nafigator.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirement-dev.txt
+          pip install -e .
```
Comment (royklip) [inline .github/workflows/github-actions-nafigator.yml:29]: requirement-dev.txt has a lot of overlap with the build dependencies. You could install only twine here and let the rest of the dependencies be installed via the "pip install -e ."

File: .github/workflows/github-actions-nafigator.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      #   with:
+      #     fileName: CHANGELOG.md
+      #     checkNotification: Simple # default `Detailed`
+      #   env:
+      #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Test with pytest
+        run: |
+          pip install pytest
+          pip install pytest-cov
```
Comment (royklip) [inline .github/workflows/github-actions-nafigator.yml:43]: These test requirements could be combined with the dev requirements (twine)

File: tests/test_highlighter.py [hunk 4]
```diff
@@ -1,80 +1,80 @@
-from nafigator.postprocessor.highlighter import Highlighter
-from lxml import etree
-import fitz
-import os
-from nafigator import NafDocument
-import unittest
+# from nafigator.postprocessor.highlighter import Highlighter
```
Comment (royklip) [inline tests/test_highlighter.py:1]: Why are these files completely commented?

File: .github/workflows/github-actions-nafigator.yml (hunk 1 above)
Comment (hmduerinck) [reply .github/workflows/github-actions-nafigator.yml:5]: check

**Comment by hmduerinck** [comment]
Wordt even ingetrokken. Nafgitor splitsing gaat door, dit wordt later terug opgepakt en vorm gegeven volgens:

https://godatadriven.com/blog/a-practical-guide-to-setuptools-and-pyproject-toml/

**Pull request closed without merging**